tests/PHPUnit/Framework/XssTesting.php

```diff
@@ -34,12 +34,19 @@ public function forTwig($type, $sanitize = false)
         return $result;
     }
 
-    // kept after vue migration for proof angularjs injection does not apply
+    /**
+     * @deprecated
+     */
     public function forAngular($type, $sanitize = false)
+    {
+        return $this->forVueJs($type, $sanitize);
+    }
+
+    public function forVueJs($type, $sanitize = false)
     {
         $n = $this->addXssEntry($type, 'angular');
```

Comment:
Any chance to also update this to `'vuejs'`?

Will require an OmniFixture rebuild, but maybe worth the effort, unless we need to keep it at the old naming for compatibility with existing (plugin) test suites?

Reply:
@mneudert I'm going to update the omni fixture in a separate PR. Will also rename that xss entry there.
So please review the PR as is for now.